config/docker/server/startBroker.sh.mustache

```diff
@@ -1,19 +1,92 @@
 #!/bin/bash
 set -e
 
+function try()
+{
+    [[ $- = *e* ]]; SAVED_OPT_E=$?
+    set +e
+}
+
+function throw()
+{
+    exit $1
+}
+
+function catch()
+{
+    export ex_code=$?
+    (( $SAVED_OPT_E )) && set +e
+    return $ex_code
+}
+
 name=$1
 mode=$2
+waitForOther=$3
+application="broker"
+theOtherApplication="server"
+config="./$application-config"
 
-echo "RUNNING startBroker.sh $name $mode"
+echo "RUNNING $application $name $mode"
+export LD_LIBRARY_PATH={{{catapultAppFolder}}}/lib:{{{catapultAppFolder}}}/deps
 
 ulimit -c unlimited
 
+rm -f "{{{dataDirectory}}}/$application.started"
+rm -f "{{{dataDirectory}}}/$application-recovery.started"
+
 if [ "$mode" == "DEBUG" ]; then
   echo "Setting up core dump..."
   mkdir -p ./logs
-  echo './logs/broker.%e.%t' > /proc/sys/kernel/core_pattern
+  echo "./logs/$application.%e.%t" >/proc/sys/kernel/core_pattern
+fi
+
+if [ "$waitForOther" == "true" ]; then
```

Comment:
Is the wait needed for the broker?

Reply:
It's not needed. I've kept it so we can potentially have a common start.sh file.